### ISSUE TITLE
[Guideline] Spelling of documentation in English

### ISSUE THREAD

**Issue by ignotus666** [jamulussoftware/jamulussoftware.github.io#147]
Reading through the documentation in English, particularly the new text that's being drafted, I see the need for an in-depth revision to correct a wide variety of mistakes (grammar, syntax, misspellings, typos...). I can volunteer to do that (if @gilgongo is ok with it) but I'd rather wait until the new changes settle in as it seems to still be an ongoing process. This would simply be a revision to correct mistakes and would not entail any changes to content, so translations would be unaffected. 

But regardless of who does the revision and before tucking into it, I think we ought to decide what spelling we're going to use: en-US or en-GB, and be consistent with it. Spelling for the GUI is in en-US - I'm not sure how consistent it actually is but Volker asked for it to be that way. As regards what to use for the documentation, I have mixed feelings - on the one hand I (and also @gilgongo if I'm not mistaken) speak and write in GB English. In my work, unless explicitly asked to do otherwise, I'll always write/proofread in en-GB. But on the other, I understand that our English-speaking audience is much wider than just the British Isles. I suspect most people wouldn't care or notice either way, but I do think there should be consistency. I recently saw a bit of text where 'minimise' is used, and then a couple of lines down it's spelled as 'minimize' - in fairness it's mainly due to occupational habit, but these things make me wince involuntarily.

I haven't done a thorough check, but I think that as @gilgongo wrote most of the documentation it's mostly in en-GB already. My personal preference would be to keep it that way, but it's not a hill I'm going to die on.

So, what are your thoughts?

**Comment by trebmuh** [comment]
I personally don't care very much which English dialect is used.
I've been in Australia for 3 years and a half, and I've seen a mix between both on a daily basis.
So I can cope with whatever English is used, even a mix.

**Comment by ignotus666** [comment]
The English language doesn't have a prescriptive authority (Spanish does, for instance) so in theory you can spell words whichever way you like. It's not a matter of dialects but of spelling consistency - picking one and sticking to it.

Even Irvine Welsh books are consistent in their spelling of Scots dialogue.

**Comment by ann0see** [comment]
I'd vote for en-GB since gilgongo wrote most of the documentation in en-GB. I don't think many people would care about that but they do care about Syntax errors, etc. Since I'm not a native speaker, I don't think I can find all the mistakes. So I'm out.

**Comment by trebmuh** [comment]
I think you misunderstood me @ignotus666 , but as said: I don't care very much of it and will cope with whatever English dialect/spelling is used.

**Comment by ignotus666** [comment]
The spelling differences I'm talking about are trivial and in no way affect one's ability to understand the text (dialects tend to differ in lexicon and grammar so that's not applicable here); all I'm proposing is to follow a consistent guideline. It has no bearing on translations or on comprehension whatsoever.

**Comment by gilgongo** [comment]
It's an interesting point!  The top country in the downloads list on SF is the US by [quite a lot](https://sourceforge.net/projects/llcon/files/stats/map?dates=2020-01-01+to+2020-12-03). Maybe 50% more than the next largest, the UK. 

So perhaps for that reason we should be American?

**Comment by ignotus666** [comment]
Having thought about it a bit more... if the people who write/maintain/correct the documentation are British/European, regardless of who the biggest audience is, I think it just makes sense for us to keep it in en-GB. Since this isn't going to lead to Americans misunderstanding anything I can't really see it being a problem. Similarly, my Spanish translation is in Castilian Spanish as it's what I speak and what I write in - and though it differs in many aspects (more so than American vs British English) to other regional variants, I doubt anyone from say, Peru, should have any trouble understanding it.

In short: since it won't affect clarity or comprehension, I think it makes most sense to write in whatever our 'native' style is; in this case, en-GB. More important than the spelling convention itself is consistency, IMHO.

**Comment by ann0see** [comment]
I'd say once the translations and #157 are finished, @ignotus666 and @gilgongo you should both look through all the .md files in english and fix mistakes. I will focus on #101 (community and move some pages there). Everything not related to grammar mistakes, style, ... should be part of a new issue.

Edit also consider  #91

**Comment by ann0see** [comment]
Any updates here? I think we should go through the whole documentation, find spelling mistakes, inconsistencies, ...

**Comment by ignotus666** [comment]
I intended to have a go at it but for the past few weeks work has left me with very little spare time. I might be able to look into it this coming week but I can't promise anything.

**Comment by gilgongo** [comment]
I'll see if I can look at it tomorrow.

**Comment by gilgongo** [comment]
@ignotus666 I think this is done now? Close?

**Comment by ignotus666** [comment]
Yeah, I'd say so.
